src/rgw/rgw_bucket.cc

```diff
@@ -1770,30 +1782,32 @@ static int process_stale_instances(rgw::sal::RGWRadosStore *store, RGWBucketAdmi
 
 int RGWBucketAdminOp::list_stale_instances(rgw::sal::RGWRadosStore *store,
                                            RGWBucketAdminOpState& op_state,
-                                           RGWFormatterFlusher& flusher)
+                                           RGWFormatterFlusher& flusher,
+                                           const DoutPrefixProvider *dpp)
 {
   auto process_f = [](const bucket_instance_ls& lst,
                       Formatter *formatter,
                       rgw::sal::RGWRadosStore*){
                      for (const auto& binfo: lst)
                        formatter->dump_string("key", binfo.bucket.get_key());
                    };
-  return process_stale_instances(store, op_state, flusher, process_f);
+  return process_stale_instances(store, op_state, flusher, dpp, process_f);
 }
 
 
 int RGWBucketAdminOp::clear_stale_instances(rgw::sal::RGWRadosStore *store,
                                             RGWBucketAdminOpState& op_state,
-                                            RGWFormatterFlusher& flusher)
+                                            RGWFormatterFlusher& flusher,
+                                            const DoutPrefixProvider *dpp)
 {
-  auto process_f = [](const bucket_instance_ls& lst,
+  auto process_f = [dpp](const bucket_instance_ls& lst,
```

Comment:
@cbodley just wanted to double check that passing `dpp` into the lambda was the proper way to get it in.